stories/__tests__/SQFormRadioButtonGroup.stories.test.js

```diff
@@ -0,0 +1,173 @@
+
+        const radioButtons = screen.getAllByRole('radio');
+
+        const giantPandaButton = radioButtons[0];
+        const giantPandaLabel = screen.getByText(/Giant Panda/i);
+
+        const redPandaButton = radioButtons[1];
+        const redPandaLabel = screen.getByText(/Red Panda/i);
+
+        const kungFuPandaButton = radioButtons[2];
+        const kungFuPandaLabel = screen.getByText(/Kung Fu Panda/i);
+
+        expect(giantPandaButton).toBeInTheDocument();
+        expect(giantPandaLabel).toBeInTheDocument();
+        expect(giantPandaLabel).toHaveTextContent('Giant Panda');
+
+        expect(redPandaButton).toBeInTheDocument();
+        expect(redPandaLabel).toBeInTheDocument();
+        expect(redPandaLabel).toHaveTextContent('Red Panda');
+
+        expect(kungFuPandaButton).toBeInTheDocument();
+        expect(kungFuPandaLabel).toBeInTheDocument();
+        expect(kungFuPandaLabel).toHaveTextContent('Kung Fu Panda');
+      });
+    });
+
+    describe('Selection', () => {
+      it('allows selecting button', () => {
+        render(<SQFormRadioButtonGroup size="auto" />);
+        const firstRadioButton = screen.getAllByRole('radio')[0];
+
+        expect(firstRadioButton).not.toBeChecked();
+        userEvent.click(firstRadioButton);
+        expect(firstRadioButton).toBeChecked();
+      });
+
+      it('allows selection via label', () => {
+        render(<SQFormRadioButtonGroup size="auto" />);
+
+        const giantPandaRadioButton = screen.getAllByRole('radio')[0];
+        const giantPandaLabel = screen.getByText(/Giant Panda/i);
+
+        expect(giantPandaRadioButton).not.toBeChecked();
+        userEvent.click(giantPandaLabel);
+        expect(giantPandaRadioButton).toBeChecked();
+      });
+
+      it('allows only one value to be selected', () => {
+        render(<SQFormRadioButtonGroup size="auto" />);
+
+        const radioButtons = screen.getAllByRole('radio');
+        const firstRadioButton = radioButtons[0];
+        const secondRadioButton = radioButtons[1];
+
+        expect(firstRadioButton).not.toBeChecked();
+        expect(secondRadioButton).not.toBeChecked();
+
+        userEvent.click(firstRadioButton);
+        expect(firstRadioButton).toBeChecked();
+        expect(secondRadioButton).not.toBeChecked();
+
+        userEvent.click(secondRadioButton);
+        expect(firstRadioButton).not.toBeChecked();
+        expect(secondRadioButton).toBeChecked();
+      });
+    });
+
+    describe('onChange handler', () => {
+      let onChangeMock;
+
+      beforeEach(() => {
+        onChangeMock = jest.fn();
+      });
+
+      it('calls onChange handler on a change', () => {
+        render(<SQFormRadioButtonGroup size="auto" onChange={onChangeMock} />);
+
+        const firstRadioButton = screen.getAllByRole('radio')[0];
+        userEvent.click(firstRadioButton);
+        expect(onChangeMock).toHaveBeenCalledTimes(1);
+      });
+    });
+  });
+
+  it('Has default value with an initialValue', () => {
```

Comment:
Did you intend for this test to be outside a describe?

Reply:
It's still inside the top level describe but I didn't even intend that. It's meant to be inside the "without validation" describe